identity/client/src/pages/tenants/detail/index.tsx

```diff
@@ -15,22 +14,18 @@ import Tabs from "src/components/tabs";
 import { DetailPageHeaderFallback } from "src/components/fallbacks";
 import Flex from "src/components/layout/Flex";
 import { useEntityModal } from "src/components/modal";
-import EditModal from "src/pages/tenants/modals/EditModal";
 import DeleteModal from "src/pages/tenants/modals/DeleteModal";
+import { spacing01, spacing03 } from "@carbon/elements";
```

Comment:
It is a good idea to keep the imports ordered when possible:
external deps imports
global imports
same directory imports

Reply:
@OmranAbazid I've thought about that before actually and I agree, so maybe we could look into adding something like [this plugin](https://www.npmjs.com/package/@trivago/prettier-plugin-sort-imports), what do you think?

This is another point I see more as fixing the issue in a project-wide fashion otherwise it's a bit pointless to change this in one file as in most other files in this repository the imports are in random order.

Reply:
I know that it is sometimes is inconsistent but we should just not introduce more tech debt.
I also think it is not worth working on this specifically and the created issue will always be deprioritized.
So every time we do a change, we try to make things a bit better.